src/Android/Xamarin.Android/Samples/Network Analysis/RouteAroundBarriers/readme.md

```diff
@@ -0,0 +1,66 @@
+# Route around barriers
+
+Find a route that reaches all stops without crossing any barriers.
+
+![Screenshot](RouteAroundBarriers.jpg)
+
+## Use case
+
+You can define barriers to avoid unsafe areas, for example flooded roads, when planning the most efficient route to evacuate a hurricane zone. You could also use this functionality to plan routes when you know an area will be inaccessible due to a community activity like a run or a market night.
+
+## How to use the sample
+
+Click 'Add stop' to add stops to the route. Click 'Add barrier' to add areas that can't be crossed by the route. Click 'Route' to find the route and display it.
+
+## How it works
+
+1. Create the route task by calling `RouteTask.CreateAsync(_serviceUrl)`.
+2. Get the default route parameters for the service by calling `_routeTask.CreateDefaultParametersAsync`.
+3. When the user adds a stop, add it to the route parameters.
+    1. Normalize the geometry; otherwise the route job would fail if the user included any stops over the international date line.
+    2. Get the name of the stop by counting the existing stops - `_stepsOverlay.Graphics.Count + 1`.
+    3. Create a composite symbol for the stop, combining a pushpin marker and a text symbol.
+    4. Create the graphic from the geometry and the symbol.
+    5. Add the graphic to the stops graphics overlay
+4. When the user adds a barrier, create a polygon barrier and add it to the route parameters.
+    1. Normalize the geometry (see **3a** above).
+    2. Buffer the geometry to create a larger barrier from the tapped point by calling `GeometryEngine.BufferGeodetic(mapLocation, 500, LinearUnits.Meters)`
+    3. Create the graphic from the geometry and the symbol.
+    4. Add the graphic to the barriers overlay.
+5. When ready to find the route, configure the route parameters.
+    1. Set the `ReturnStops` and `ReturnDirections` to true
+    2. Create a `Stop` for each graphic in the stops graphics overlay. Add that stop to a list, then call `_routeParameters.SetStops(routeStops)`.
+    3. Create a `PolygonBarrier` for each graphic in the barriers graphics overlay. Add that barrier to a list, then call `_routeParameters.SetPolygonBarriers(routeBarriers)`.
+    4. If the user will accept routes with the stops in any order, set `FindBestSequence` to true to find the most optimal route.
```

Comment:
Should the values be formatted to make them a bit more readable? 
true => `true`